### ISSUE TITLE
Corrections to Existing Hook Scripts 

### ISSUE THREAD

**Issue by jaxinewolfe** [Smithsonian/CCN-Data-Library#115]
**Running list:**

_Spot correction for Kauffman depths:_
  mutate(
depth_min = case_when(study_id == "Kauffman_et_al_2020_Brazil" & depth_max == 15 ~ 0,
                          study_id == "Kauffman_et_al_2020_Brazil" & depth_max == 30 ~ 15,
                          study_id == "Kauffman_et_al_2020_Brazil" & depth_max == 50 ~ 30,
                          study_id == "Kauffman_et_al_2020_Brazil" & depth_max == 100 ~ 50,
                          study_id == "Kauffman_et_al_2020_Brazil" & depth_max == 300 ~ 100,
                          T ~ depth_min))

_Make Kemp core IDs more unique_
study_id == "Kemp_et_al_2024" ~ paste(site_id, core_id, sep = "_")

**Comment by jaxinewolfe** [comment]
@cheneyr one more for you:

- [ ] Kusumaningtyas_et_al_2018 has some carbon isotope values that didn't need to be converted - can you drop line 241 in the hook script?

I'll take care of the Kauffman_et_al_2020_Brazil

**Comment by cheneyr** [comment]
> @cheneyr one more for you:
> 
> * [x]  Kusumaningtyas_et_al_2018 has some carbon isotope values that didn't need to be converted - can you drop line 241 in the hook script?
> 
> I'll take care of the Kauffman_et_al_2020_Brazil

Yes will do! Which Kauffman_et_al_2020_Brazil do we need to fix? I found the same study_id in some of the CIFOR data and the Maxwell synthesis?

**Comment by jaxinewolfe** [comment]
YET TO BE ADDRESSED

- [ ] investigate Nahlik and Fennessy core NWCA11-1368-4970 has no depth max?
- [ ] Boyd_et_al_2017 - there is a depth issue that needs to be fixed in the data release 
- [ ] no depths for Langston_et_al_2022 for agedepth assessments that span the whole core
- [ ] no depth max for intervals in Smeaton_et_al_2022a and Sammul_et_al_2012, Quafisi_et_al_2024, and Trettin_et_al_2017
- [ ] publication_type needs standardization across the synthesis
- [ ] Sharma et al 2021 has a mangrove coord way out in the ocean (Koh_Kohng_133_364)
- [ ] Update Royna (Barten CIFOR) DOI: 10.17528/CIFOR/DATA.00305
- [ ] Nolte_2020 core "NFB_ungrazed_seaward" has a depth interval mismatch starting at the end of core "NFB_ungrazed_landward". Occurs in the original data contributed, so needs correction in the data release. 
- [ ] Check Sanderman for missing dry bulk density and why
- [ ] Add Trettin et al 2020 associated article: Trettin, Carl C.; Dai, Zhaohua; Tang, Wenwu; Lagomasino, David; Thomas, Nathan; Lee, Seung Kuk; Simard, Marc; Ebanega, Medard Obiang; Stoval, Atticus; Fatoyinbo, Temilola E. 2021. Mangrove carbon stocks in Pongara National Park, Gabon. Estuaries, Coastal and Shelf Science. 259(11): 107432. https://doi.org/10.1016/j.ecss.2021.107432
- [ ] Add reference for Trettin 2017: https://doi.org/10.1016/j.foreco.2015.06.027
